File: C++/Edwin Villarraga/matematica/CassiniIdentity.cpp
#include<bits/stdc++.h>
#include<cstdlib>
using namespace std;
int cassini(int n){
    return (n&1)!=0? -1 : 1;
}
int main(){
    int n=5;
    printf("%d\n",cassini(n));
}

File: Codigos estudiantes por lenguaje/Java/Edwin Villarraga/matematicas/CassiniIdentity.java
public class Main {
	    static int cassini(int n) {
	        return (n & 1) != 0 ? -1 : 1;
	    }

	    public static void main(String args[]) {
	        int n = 5;
	        System.out.println(cassini(n));
	   }
}
